sdk/src/main/scala/com/horizen/storage/SidechainSecretStorage.scala

```diff
@@ -44,20 +45,26 @@ class SidechainSecretStorage(storage: Storage, sidechainSecretsCompanion: Sidech
 
   def getAll: List[SidechainTypes#SCS] = secrets.values.toList
 
+  private def nextVersion: Array[Byte] = {
+    val version = new Array[Byte](32)
+    Random.nextBytes(version)
+    version
+  }
+
   def add (secret: SidechainTypes#SCS): Try[SidechainSecretStorage] = Try {
     require(secret != null, "Secret must be NOT NULL.")
-    val version = new Array[Byte](32)
+
     val key = calculateKey(secret.publicImage())
 
     require(!secrets.contains(key), "Key already exists - " + secret)
 
     val value = new ByteArrayWrapper(sidechainSecretsCompanion.toBytes(secret))
 
-    scala.util.Random.nextBytes(version)
-
+    val version = nextVersion
     storage.update(new ByteArrayWrapper(version),
       List(new JPair(key, value)).asJava,
       List[ByteArrayWrapper]().asJava)
+    log.debug("Sidechain secret storage updated with version: " + new ByteArrayWrapper(version))
```

Comment:
remove logs.
In general, remove all recently added logs from every storage class. There is no sense to keep them anymore.